### PR TITLE
Add densenet and cspnet example

### PR DESCRIPTION
**Pull request by Varchita-Beena**
The example is about DenseNet, CSPNet their varaints on Cifar10 dataset

### CODE REVIEW

**Comment by Varchita-Beena** [comment]
> @googlebot I signed it!

@googlebot I signed it!

File: examples/vision/densenet_cspnet_variants.py [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+"""
+"""shell
+densenet_cspnet_variants.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1ejnjgRzfo60FWXMp8_HyzQJ_5kBLIdwd
+
+##DenseNet
```
Comment (fchollet) [inline examples/vision/densenet_cspnet_variants.py:16]: The example should explain the point of these models -- why should a reader be interested in them? How should they use them? How do they compare to alternatives?

The example will also need to be polished much further, as it currently has many issues. Please check out README instructions for how to develop and preview code examples.

File: examples/vision/densenet_cspnet_variants.py [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+    BatchNormalization,
+    Reshape,
+    Flatten,
+)
+from tensorflow.keras.optimizers import Adam
+from tensorflow.keras.losses import CategoricalCrossentropy
+from tensorflow.keras import backend as K
+from tensorflow.keras.utils import to_categorical
+import matplotlib.pyplot as plt
+from sklearn.model_selection import train_test_split as split
```
Comment (xybrs) [inline examples/vision/densenet_cspnet_variants.py:99]: sklearn is unnecessary dependency in this example, we can use split from numpy

**Comment by pcoet** [comment]
Hi @Varchita-Beena, thanks for this pull request. Would you like to update the example, based on the feedback? Otherwise we'll close the request.

**Comment by Varchita-Beena** [comment]
Yes I would like to update the example based on the feedback. I will complete it soon.
Thanks. 
 
  On Mon, Aug 14, 2023 at 8:30 AM, David ***@***.***> wrote:   


Hi @Varchita-Beena, thanks for this pull request. Would you like to update the example, based on the feedback? Otherwise we'll close the request.

**Pull request closed without merging**